pkg/release/release_version_test.go

```diff
@@ -85,14 +85,14 @@ func TestSetReleaseVersion(t *testing.T) {
 			// new beta from beta
 			releaseType:  release.ReleaseTypeBeta,
 			version:      "v1.18.4-beta.1.2+3ff09514d162b0",
-			branch:       "release-1.18",
-			parentBranch: "release-1.18",
+			branch:       git.Master,
```

Comment:
Quick sanity check for myself here: is this `git.Master` here since we are cutting the release branches later now?

Reply:
Yes, I had to actually fix the logic. Luckily it has not used anywhere till now. :)